spiped/main.c

```diff
@@ -302,4 +303,18 @@ main(int argc, char * argv[])
 	 * If we could reach this point, we would free memory, close sockets,
```

Comment:
The `/* NOTREACHED */` is correct since we never fall out of the loop; all we ever do is jump to error-handling.